application/gqlgen/user.go

```diff
@@ -171,7 +167,7 @@ func (r *queryResolver) User(ctx context.Context, id *string) (*models.User, err
 
 	dbUser := models.User{}
 	selectFields := GetSelectFieldsForUsers(ctx)
```

Comment:
`selectFields` can be removed here if you don't need the `extras` below. Then `GetSelectFieldsForUsers` and `UserFields` can go away as well.